unstructured_ingest/connector/nomicdb.py

```diff
@@ -0,0 +1,125 @@
+import multiprocessing as mp
```

Comment:
For new connectors, this should live in the v2 directory using the new ingest framework: [unstructured_ingest/v2/processes/connectors](https://github.com/Unstructured-IO/unstructured-ingest/tree/main/unstructured_ingest/v2/processes/connectors)

Reply:
This will change the general approach you've introduced here so I'll wait to review the actual connector files until that's been updated.